### PR TITLE
Make Auth Redux and NavBar

### PR DESCRIPTION
**Pull request by Satyam-Mechlintech**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,26 +1,38 @@
...
+  import React, { useState } from "react";
+  import { baseURL } from './env';
+  import axios from 'axios';
+  import AdminPage from './pages/AdminPage';
+  import Carousel from './components/Carousel';
 
+  function App() {
+  // const axiosInstance=axios({baseURL:"http://localhost:8080/api"})
+  // axios.interceptors.request.use((request)=> request)
+  // axios.interceptors.response.use((response)=> response,(err)=>{
```
Comment (shivshankar12345) [inline src/App.tsx:18]: Remove commented codes.

File: src/Redux/store.ts [hunk 2]
```diff
@@ -21,4 +21,9 @@ export const persister=persistStore(store)
 // // Infer the `RootState` and `AppDispatch` types from the store itself
 // export type RootState = ReturnType<typeof store.getState>
 // // Inferred type: {posts: PostsState, comments: CommentsState, users: UsersState}
-// export type AppDispatch = typeof store.dispatch
\ No newline at end of file
+// export type AppDispatch = typeof store.dispatch
+
```
Comment (shivshankar12345) [inline src/Redux/store.ts:25]: Remove the comments

File: src/components/Carousel.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+//   useEffect(() => {
+//     const fetchImages = async () => {
+//       try {
+//         const response = await axios.get("your-backend-endpoint");
+//         setImages(response.data); // Assuming the data is an array of image URLs
+//       } catch (error) {
+//         console.error("Error fetching images:", error);
+//       }
+//     };
+
```
Comment (shivshankar12345) [inline src/components/Carousel.tsx:17]: Same

File: src/components/Carousel.tsx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+//     fetchImages();
+//   }, []);
+  // Static array of image URLs for now
+  const images = [
+    "https://via.placeholder.com/1920x1080?text=Image+1",
+    "https://via.placeholder.com/1920x1080?text=Image+2",
+    "https://via.placeholder.com/1920x1080?text=Image+3",
+  ];
```
Comment (shivshankar12345) [inline src/components/Carousel.tsx:25]: Add static images in single utils files.

File: src/types/login.types.ts [hunk 5]
```diff
@@ -7,4 +7,10 @@ export interface IOTPInput {
 export interface IntialState{
     accessToken:string;
     refreshToken:string,
-}
\ No newline at end of file
+}
+
+
```
Comment (shivshankar12345) [inline src/types/login.types.ts:12]: Same

**Review by shivshankar12345 — CHANGES_REQUESTED** [review]

File: src/Redux/store.ts (hunk 2 above)
Comment (Satyam-Mechlintech) [reply src/Redux/store.ts]: Remove Comments

**Comment by Satyam-Mechlintech** [comment]
Remove All the Comments

**Review by shivshankar12345 — APPROVED** [review]

**Pull request merged**